docs/paper/dev/misc/databases.mdx

```diff
@@ -0,0 +1,208 @@
+---
+slug: dev/using-databases
+---
+
+# Using Databases
+
+When you are storing larger amounts of data inside a plugin, we recommend using a database. This guide will walk you through the startup process.
+
+## What is a database?
+
+A database is a collection of information that is stored electronically on a computer system. There are many different types of databases,
+and the main two are SQL and NoSQL.
+
+### NoSQL vs SQL
+
+A NoSQL (Not Only SQL) database is a type of database management system that diverges from the traditional relational database model. 
+Unlike traditional SQL databases, which store data in structured tables with predefined schemas, NoSQL databases are schema-less 
+and offer flexible data models. 
+
+They are designed to handle large volumes of unstructured or semi-structured data. 
+NoSQL databases use various data models, such as key-value, document, column-family, or graph, depending on 
+the specific requirements of the application.
+
+On the other hand, An SQL database is a type of database management system that follows the relational database model. 
+It organizes data into structured tables with predefined schemas, where each table represents an entity and columns 
+represent attributes of that entity. SQL (Structured Query Language) is used to interact with the database, 
+allowing users to perform various operations like querying, inserting, updating, and deleting data.
+
+## File-based vs Standalone Databases
+
+When working with databases, you have two options: file-based or standalone. File-based databases are stored in a file on the disk,
+and are usually used for smaller databases. Standalone databases are stored in a separate process, and are usually used for larger databases.
+
+### File-based Databases
+
+File-based databases are all stored within a single file on the disk. They are usually used for smaller databases, as they are easier to set up and use.
+They can be created and handled from within your plugin code, but offer lesser performance than standalone databases.
+Some examples of file-based databases are `SQLite` and `H2`.
+
+<details>
+    <summary>Simple SQLite Setup</summary>
+
+## SQLite
+
+To work with SQLite you will need a driver to connect / initialise the database. 
+We will be recommending the JDBC connector which can be added to your project with the following dependency:
```

Comment:
The dependency declaration isn't useful because calling classes in `org.sqlite` packages is unnecessary. No dependency is needed; the whole interface works seamlessly with JDBC.

Reply:
Good point actually, this is only needed if you intend to relocate a different version of JDBC